config.rb

```diff
@@ -89,9 +89,22 @@ def page_description(page=current_page, locale=I18n.locale)
     t("head.default_description") || ""
   end
 
-  # Body classes
-  def page_classes(path=current_path.dup, options={})
-    super(path.sub(/^[a-z]{2}\//, ''), options)
+  # Page body classes
+  def page_classes(path = current_path.dup, options = {})
+    # Prevent page classes from being translated
```

Comment:
Is this part coming from the Defacto website? Just wondering because this might not work as expected for Middleman v4. But you probably tested this already?